overlord/ifacestate/ifacestate_test.go

```diff
@@ -395,7 +395,7 @@ func (s *interfaceManagerSuite) TestSmokeAppArmorPromptingEnabled(c *C) {
 	c.Check(stopCount, Equals, 0)
 	mgr.Stop()
 	c.Check(stopCount, Equals, 1)
-	c.Check(mgr.InterfacesRequestsManager(), IsNil)
+	c.Check(mgr.InterfacesRequestsManager() == nil, Equals, true)
```

Comment:
if so, we have a checker for that `IsInterfaceNil`

Reply:
I forgot about that

Reply:
changed